common/SC_ReplyImpl.hpp

```diff
@@ -25,7 +25,14 @@
 
 #include <boost/asio.hpp>
 
-enum Protocol { kUDP, kTCP };
+enum Protocol {
+    kUDP,
+    kTCP
+#ifdef __EMSCRIPTEN__
+    ,
+    kWeb
+#endif
```

Comment:
can we have documentation for kWeb protocols? What does this look like? Limitations?